### DISCUSSION TITLE
Problema HA, dopo alcune ore non è più raggiungibile

### DISCUSSION THREAD

**Discussion by lucaweb78** [indomus/forum#2448] [HUB: Home Assistant]
Ciao a tutti, come da titolo da circa una settimana, dopo alcune ore di funzionamento e/o comunque senza logica (magari funziona per un giorno) il sistema non è più raggiungibile, né da rete interna né da rete esterna. Unica alternativa riavviare il minipc. 
Avevo un pc con ha aperto e mi diceva di ricaricare l'interfaccia. 
Mi fa così circa da una settimana. Prima di allora nessun problema. 
Ho installato Glances ma è sempre tutto fin troppo nella norma con percentuali bassissime e/o valori mai in rosso. 
Esiste un log che posso controllare? Una sorta di event viewer? 
Grazie mille a tutti.

**Comment by indomus** [comment]
non è che banalmente ti va in standby il minipc

**Reply by lucaweb78** [reply]
bella risposta. Non credo ma come potrei verificarlo? E' un beelink e ha solo HA installato

**Comment by indomus** [comment]
se lo hai installato su debian, così: https://indomus.it/guide/le-prime-cose-da-fare-dopo-aver-installato-linux-debian-11-per-la-propria-domotica/

**Reply by lucaweb78** [reply]
grazie e scusa l'ignoranza. Ma se all'avvio parte direttamente ha, come lo lancio quel comando? Posso farlo da dentro ha andando su Terminal?

**Reply by lucaweb78** [reply]
e comunque ha funzionato benissimo per oltre un mese, poi ha cominciato a fare così, direi che non è lo standby

**Reply by indomus** [reply]
> grazie e scusa l'ignoranza. Ma se all'avvio parte direttamente ha, come lo lancio quel comando? Posso farlo da dentro ha andando su Terminal?

Stai usando HassOS, non Core@Docker@Debian. fai conto che non ti sia stato suggerito quanto sopra.
devi capire diversamente qual è il problema (il primo è proprio usare HassOS, ma vabbè)

**Reply by lucaweb78** [reply]
sarebbe bello sapere come capirlo. Si è di nuovo bloccato. Boh

**Comment by lucaweb78** [comment]
Niente da ieri sera nonostante il minipc sia acceso (o riavviato) e anche la lan lampeggia e nel router vedo l'ip correttamente attivo non si avvia HassOs. Soluzioni? Alternative? i backup li ho!

**Reply by KC-inDomus** [reply]
collega un monitor. che ti dice a schermo
